cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -91,6 +91,39 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           });
         });
 
+        if (service === 'news') {
+          it('should render a visible guidance message', () => {
+            cy.window().then(win => {
+              const aresMedia = getBlockData(
```

Comment:
At this stage, this is just the media (video) block and not Ares data. Please can we rename this variable to `media` to improve readability and maintain consistency with our other tests?

Reply:
Addressed and fixed